Containerd/self-assessment.md

```diff
@@ -0,0 +1,350 @@
+
+## Appendix
+
+
+* Known Issues Over Time:
+  
+   There have been some problems in the past with the plugins that containerd has. Even though it is a feature, it has led to problems when the plugins are not correctly inputted.
+    - https://github.com/containerd/containerd/pull/7347
+    - https://github.com/containerd/containerd/pull/8056
+
+  There is also a few issues with its ability to access resources. In attempt to keep least-privilege access, the dulling out when available can be problematic
+    - https://github.com/containerd/containerd/issues/3351
+    - https://www.cvedetails.com/cve/CVE-2023-25153/
+    - https://www.cvedetails.com/cve/CVE-2022-31030/
+    - https://www.cvedetails.com/cve/CVE-2022-23471/
+    - https://www.cvedetails.com/cve/CVE-2021-32760/
+  
+
+* Record in catching issues in code review or automated testing:
+  
+   **Current Level: [![OpenSSF Best Practices](https://www.bestpractices.dev/projects/1271/badge)](https://www.bestpractices.dev/projects/1271)**
+  
+   The project achieved the following
+     - Basics: 13/13 passed
+     - Change Control: 9/9 passed
+     - Reporting: 8/8 passed
+     - Quality: 13/13 passed
+     - Security: 16/16 passed
+     - Analysis: 8/8 passed
+
+
+* Case Studies:
+  
+   Demonstrates how Red Hat OpenShift, integrated with containerd, streamlines containerization adoption and simplifies Kubernetes management.
+  
+   https://swapnasagarpradhan.medium.com/install-a-kubernetes-cluster-on-rhel8-with-conatinerd-b48b9257877a
+
+  Explores how containerd simplifies container management on Google Kubernetes Engine (GKE), Google Cloud's fully managed Kubernetes service.
+
+  https://cloud.google.com/kubernetes-engine
+
+  Delves into the integration of containerd with Amazon Elastic Container Service (ECS), Amazon Web Services' container orchestration service
+
+  https://aws.amazon.com/blogs/containers/tag/containerd/
+
+  Explores how containerd enables organizations to effectively manage containers on Azure Kubernetes Service (AKS), Microsoft Azure's managed Kubernetes service
+
+  https://azure.microsoft.com/en-us/updates/generally-available-containerd-support-for-windows-in-aks/
+  
+* Related Projects / Vendors:
```

Comment:
Please elaborate how these project/vendors relate to this project, and what the key differences between them are.